flow/flow.go

```diff
@@ -309,28 +310,43 @@ func (p *Packet) TransportFlow() (gopacket.Flow, error) {
 	return layer.TransportFlow(), nil
 }
 
+//Hash flow with custom func
+func Hash(f gopacket.Flow, hasher hash.Hash) {
+	if strings.Compare(string(f.Src().Raw()), string(f.Dst().Raw())) > 0 {
```

Comment:
may retrieve the the endpoints only once with src, dst := f.Endpoints()

Reply:
done